### PR TITLE
fix: modify build error in humble

### PR DESCRIPTION
**Pull request by wep21**
## Types of PR

- [ ] New Features
- [ ] Upgrade of existing features
- [x] Bugfix

## How to review this PR.
confirm if lanelet2_extension_psim can be built on both galactic and humble.

### CODE REVIEW

**Review by kenji-miyake — COMMENTED** [review]
The changes look good to me, but as I'm not a maintainer, I'll leave it to Kataoka-san.

**Comment by wep21** [comment]
@piotr-zyskowski-rai If you still face the build issue on humble, could you try this branch?
You can try to test like
```
git fetch origin
git checkout fix/build-error-humble
git merge origin/feature/zmqpp_vendor
```

**Comment by wep21** [comment]
@hakuturu583 

> so please also update [release note](https://github.com/tier4/scenario_simulator_v2/blob/master/docs/ReleaseNotes.md).

I updated the release note in https://github.com/tier4/scenario_simulator_v2/pull/793/commits/171a4c476e0311d24e070e1ee3e2034a2c499be3.

**Review by yamacir-kit — APPROVED** [review]

**Pull request merged**
